### ISSUE TITLE
Fix deploy step in CI failing

### ISSUE THREAD

**Issue by e00E** [gnosis/dex-services#958]
The new rust version introduced more warnings or lints that make master
fail in the ethcontract generated code. https://travis-ci.com/github/gnosis/dex-services/jobs/343998701

https://github.com/gnosis/dex-services/pull/957

**Comment by e00E** [comment]
Description above can't be fully correct because it doesn't happen locally.

**Comment by e00E** [comment]
It is weird that CI is succeeding for PRs but not for master. It must be related to a difference in the jobs we run. I noticed that we have 
```
      cache:
        directories:
          - $HOME/.cache/yarn
          - $HOME/.cargo/bin
      before_cache:
        - rm -rf "$TRAVIS_HOME/.cargo/registry/src"
      env: CARGO_INCREMENTAL=0
```
only for PRs but not for deploy.

**Comment by nlordell** [comment]
So, If figured out what was going on. We are getting a warning:
```
warning: unnecessary braces around block return value
```

This is caused because `ethcontract` generates the following code for the `lazy_static` `Artifact`:
```rust
lazy_static! {
    pub static ref ARTIFACT: Artifact = { Artifact::from_json("...") };
}
```

This complains that the `{ ... }` braces aren't needed (which is true).

This is only an issue with the "Deploy" step because we don't install `rustfmt`. As part of the standard build step, `rustfmt` is being installed and automatically used to format the generated Rust code. It is rewriting the above statement as:

```rust
lazy_static! {
    pub static ref ARTIFACT: Artifact = Artifact::from_json("...");
}
```

Which produces no warning.

Here are some of the options I can think of to solve this:
1. Leave as-is and wait until the next `ethcontract` release - it doesn't have this issue anymore as the generated code is slightly different.
2. Make a `0.7.3` patch for `ethcontract-generate` that fixes this issue. I think this is a lot of work for something so inconsequential.
3. Make a `v0.7.2-rust1.44` branch and use a Cargo patch pointing to this branch of the git repo to temporarily fix the issue.
4. Install `rustfmt` for the deploy step

**Comment by e00E** [comment]
I like staying on rust version 1.43.0 until ethcontract is released because I expect it won't take long until then. No need for more complex workarounds.

**Comment by nlordell** [comment]
This was fixed by #1214
